shared/common-adapters/section-list.desktop.tsx

```diff
@@ -234,15 +236,12 @@ class SectionList extends React.Component<Props, State> {
       <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} style={styles.container}>
         {this.props.disableAbsoluteStickyHeader && stickyHeader}
         <Kb.ScrollView
-          style={Styles.collapseStyles([styles.scroll, this.props.style])}
+          style={Styles.collapseStyles([styles.scroll, this.props.contentContainerStyle])}
           onScroll={this._onScroll}
         >
           {renderElementOrComponentOrNot(this.props.ListHeaderComponent)}
-          {/*
-          // @ts-ignore */}
           <ReactList
-            itemRenderer={this._itemRenderer as any}
-            itemSizeEstimator={this.props.itemSizeEstimator}
```

Comment:
we need the item size estimator, why remove

Reply:
I think I removed all these because nothing was passing an `itemSizeEstimator` to `Kb.SectionList` so I assumed we didn't need it.